readthedocs/projects/tasks/mixins.py

```diff
@@ -44,10 +44,17 @@ def sync_versions(self, vcs_repository):
         # check this here and do not depend on ``vcs_repository``.
         sync_tags = not self.data.project.has_feature(Feature.SKIP_SYNC_TAGS)
         sync_branches = not self.data.project.has_feature(Feature.SKIP_SYNC_BRANCHES)
-        branches, tags = vcs_repository.lsremote(
-            include_tags=sync_tags,
-            include_branches=sync_branches,
-        )
+        try:
+            branches, tags = vcs_repository.lsremote(
+                include_tags=sync_tags,
+                include_branches=sync_branches,
+            )
+        except RepositoryError:
+            log.warning(
+                "Error getting versions from the repository.",
```

Comment:
```suggestion
                "Error running lsremote to get versions from the repository.",
```

I'd like to mention `lsremote` in the message to make it explicit.